### ISSUE TITLE
Provide fall back solutions for browsers that don't support card unicode characters

### ISSUE THREAD

**Issue by raatmarien** [raatmarien/playcardswith.me#116]
Maybe show text or an image instead?

**Comment by flori9** [comment]
Another option would be to use a webfont with these characters.
The following fonts support these: https://www.fileformat.info/info/unicode/block/playing_cards/fontsupport.htm (there are probably more)
From the free fonts, Symbola Regular works the best, although it does not fit on the card well.

The best option would probably be to just use images; this gives the most consistent and beautiful result.

**Comment by raatmarien** [comment]
Good point about webfonts. I agree that images would probably be best. It gives the most flexibility and we can customize the styling best. Only downside is that images need to be made or found somewhere. I'm not sure if their are beautiful free card images online, but maybe there are. Anyway, before we start putting effort into making any ourselves, we should commit to one aspect ratio for the cards.
